<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCharacterEditor</class>
 <widget class="QMainWindow" name="CCharacterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prime World Editor - Character Editor</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="SkeletonHierarchyTreeView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="indentation">
        <number>15</number>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
        <item>
         <widget class="CCharacterEditorViewport" name="Viewport" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="AnimSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="LoopButton">
            <property name="text">
             <string>Loop</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RewindButton">
            <property name="text">
             <string>Rewind</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PlayPauseButton">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="FastForwardButton">
            <property name="text">
             <string>Fast Forward</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="AnimSpeedLabel">
            <property name="text">
             <string>Speed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="WDraggableSpinBox" name="AnimSpeedSpinBox">
            <property name="suffix">
             <string>x</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="FrameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Frame 0 / 0 (0.000s/0.000s)</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionOpen"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Animation</string>
    </property>
    <addaction name="ActionShowGrid"/>
    <addaction name="ActionShowMesh"/>
    <addaction name="ActionShowSkeleton"/>
    <addaction name="ActionBindPose"/>
    <addaction name="ActionOrbit"/>
    <addaction name="separator"/>
    <addaction name="ActionPlay"/>
    <addaction name="ActionRewind"/>
    <addaction name="ActionFastForward"/>
    <addaction name="separator"/>
    <addaction name="ActionPrevAnim"/>
    <addaction name="ActionNextAnim"/>
   </widget>
   <addaction name="MenuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionOpen"/>
   <addaction name="separator"/>
   <addaction name="ActionShowGrid"/>
   <addaction name="ActionShowMesh"/>
   <addaction name="ActionShowSkeleton"/>
   <addaction name="ActionBindPose"/>
   <addaction name="ActionOrbit"/>
  </widget>
  <action name="ActionOpen">
   <property name="icon">
    <iconset resource="../Icons.qrc">
     <normaloff>:/icons/Open.png</normaloff>:/icons/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ActionShowSkeleton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Skeleton</string>
   </property>
   <property name="toolTip">
    <string>Show Skeleton</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="ActionShowMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Mesh</string>
   </property>
   <property name="toolTip">
    <string>Show Mesh</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="ActionBindPose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bind Pose</string>
   </property>
   <property name="toolTip">
    <string>Bind Pose</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="ActionOrbit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Orbit</string>
   </property>
   <property name="toolTip">
    <string>Orbit</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="ActionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Icons.qrc">
     <normaloff>:/icons/GridLight.png</normaloff>:/icons/GridLight.png</iconset>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="ActionPrevAnim">
   <property name="text">
    <string>Previous Anim</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="ActionNextAnim">
   <property name="text">
    <string>Next Anim</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="ActionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="ActionRewind">
   <property name="text">
    <string>Rewind</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="ActionFastForward">
   <property name="text">
    <string>Fast Forward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCharacterEditorViewport</class>
   <extends>QWidget</extends>
   <header>Editor/CharacterEditor/CCharacterEditorViewport.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WDraggableSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>Editor/Widgets/WDraggableSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
